.xo-config.js

```diff
@@ -50,7 +43,14 @@ module.exports = {
 				svg: 'always'
 			}
 		],
-		'unicorn/no-fn-reference-in-iterator': 'off'
+
+		// Temporarily disabled
+		'@typescript-eslint/no-invalid-void-type': 'off',
```

Comment:
The rule recommends not using `void` in a union type, but `undefined | false` didn't seem to work, so not entirely sure what the correct solution is.

Reply:
> but `undefined | false` didn't seem to work

That might be a bug then, because their documentation specifically talks about returning undefined. 🤔

Reply:
Either way I don't particularly want to clutter each `init` function with `return undefined`

Reply:
I agree. We can leave it off for now, and we can think about whether it should be disabled in XO or just here. I haven't made up my mind yet.